style.css

```diff
@@ -0,0 +1,279 @@
+* {
+    box-sizing: border-box;
+}
+body {
+    margin: 0px;
+    font-family: 'Epilogue', sans-serif;
+}
+nav {
+    display: flex;
+    justify-content: space-between;
+    margin: 20px 15px;
+    position: relative;
+}
+
+.img-container {
+    width: 100vw;
+}
+.img-container img {
+    width: 100%;
+}
+.main-container {
+    display: flex;
+    flex-direction: column-reverse;
+    text-align: center;
+}
+.burger-nav-button {
+    padding: 0px;
+    border: none;
+    background-color: transparent;
+    cursor: pointer;
+}
+.left-container button {
+    border: none;
+    padding: 15px 25px;
+    background-color: #151515;
+    color: #ffffff;
+    border-radius: 15px;
+    font-size: 18px;
+    margin-bottom: 50px;
+}
+.left-container {
+    margin: 20px;
+}
+.left-container p {
+    margin-bottom: 30px;
+    color: #676767;
+    line-height: 150%;
+}
+.logos {
+    display: flex;
+    align-items: center;
+    gap: 30px;
+}
+.img-container {
+    width: 100%;
+}
+.mobile-display {
+    position: absolute;
+    right: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    background-color: rgb(255, 255, 255);    
+    top: 0px;
+    padding-top: 20px;
+    z-index: 3;
+    height: 100vh; 
+}
+.mobile-drop-down > button {
+    padding-left: 20px;
+}
+.mobile-drop-down {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+}
+.side-nav button {
+    border: none;
+    background-color: transparent;
+    display: flex;
+    align-items: center;
+    gap: 30px 20px;
+}
+.mobile-display .burger-nav-button { 
+    padding-right: 15px;  
+}
+.side-nav {
+    width: 66vw;
+    max-width: 250px;
+    padding-left: 20px;
+    padding-top: 30px;
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+}
+.overlay {
+    position: fixed; 
+    width: 100vw; 
+    height: 100vh; 
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(0,0,0,0.75); 
+    z-index: 2;
+}
+.hidden {
+    display: none;
+}
+.nav-icon-container {
+    height: 17px;
+    width: 17px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.desktop-display {
+    display: none;
+}
+
+.desktop-display > .buttons > .left > .desktop-menu-item-container {
+    display: flex;
+    align-items: center;
+    position: relative;
+    cursor: pointer;
+}
+
+.desktop-display > .buttons > .left > .desktop-menu-item-container .submenu {
+    display: none;
+    position: absolute;
+    background-color: blueviolet;
+    bottom: 0;
+    transform: translateY(calc(100%));
+    padding: 25px;
+}
+.desktop-display > .buttons > .left > .desktop-menu-item-container .features {
+    right: 0;
+}
+
+.desktop-display > .buttons > .left > .desktop-menu-item-container:hover .submenu {
+    display: flex;
+    flex-direction: column;
+    gap: 15px;
+    background-color: white!important;
```

Comment:
important is not necessary to use here, so you can remove it.

The :hover selector is more specific and also comes after the previous background-color rule so it will override it